### DISCUSSION TITLE
Discrepancy between the Storage usage at Github.com vs GitHub CLI

### DISCUSSION THREAD

**Discussion by sumitgupta-stx** [community/community#66954] [Enterprise]
I have an enterprise plan and we are receiving warnings about Storage usage under "Billing and plans" that says we have used all 50GB of our storage space. But when I use GitHub CLI and run:

gh repo list <Org Name> --limit 99999 --json name,diskUsage

It gives me sizes of individual repos and I add it up, then the size is well over 100GB.  What will explain this discrepancy? Is GitHub CLI reporting uncompressed size?

**Comment by queenofcorgis** [comment]
The `Shared storage` you are seeing alerts for consists of Actions artifacts and Packages usage, it is not related to [repository size limits](https://docs.github.com/repositories/working-with-files/managing-large-files/about-large-files-on-github#repository-size-limits). 

An Enterprise owner can download a detailed usage report, here are the steps: 

[Viewing GitHub Packages usage for your enterprise account](https://docs.github.com/enterprise-cloud@latest/billing/managing-billing-for-github-packages/viewing-your-github-packages-usage#viewing-github-packages-usage-for-your-enterprise-account)

The storage calculation is based on an hourly aggregate of use. Costs are based on size and how long it's stored, and not just size alone. For example, If I upload 3 GB and delete it after 3 days, I'll have accumulated 72 hours worth of usage costs for that 3 GB.

Read more about how storage is calculated here:

[Sample storage cost calculation](https://docs.github.com/enterprise-cloud@latest/billing/managing-billing-for-github-actions/about-billing-for-github-actions#sample-storage-cost-calculation)

It's worth noting that the warning you see may be related to the `Included services`, which is the free quota of Actions minutes and storage that's included in your billing plan. For the Enterprise plan, that's 50,000 Actions minutes and 50GB of storage. Team plan, that's 3,000 Actions minutes and 2GB of storage.

When you've hit the top of that limit, we put up a message and send an email notification to let you know you'll be going into overages and can expect to see an extra charge on your next payment, but no further action is required.

Your monthly Actions minutes and bandwidth usage resets each month on the billing date. Existing storage will carry over to the next month if the objects remain stored in the account. 

If you find that you've exhausted your monthly included usage and you'd like to continue to use GitHub Actions and Packages between now and your next billing date, you can increase the spending limit. For assistance beyond this, I recommend reaching out to your sales representative.
